src/tools/kata-ctl/Cargo.toml

```diff
@@ -15,7 +18,9 @@ clap = { version = "3.2.20", features = ["derive", "cargo"] }
 serde_json = "1.0.85"
 thiserror = "1.0.35"
 
-# See: https://github.com/kata-containers/kata-containers/issues/5438
+[target.'cfg(target_arch = "s390x")'.dependencies]
```

Comment:
Is it possible to let both s390x and others use `native-tls`?

Reply:
@jodh-intel ran into issues with openssl and musl. s390x uses libc instead. I guess, ppc would be same as s390x; not sure about aarch64.

Reply:
@liubin - as @hbrueckner is finding, musl is the problem.

We could just link to glibc to allow native TLS since `kata-ctl` isn't running inside the container. But if we do that, we should link consistently to the same libc for all host-side elements imho (including runtime-rs/dragonball, trace-forwarder, agent-ctl, _etc_). But that's a bigger topic we should probably discuss at the AC meeting.

Reply:
:+1: to have this as a follow-up discussion.